### PR TITLE
cipher: edit page

### PR DESCRIPTION
**Pull request by lenaArtinger**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/windows/cipher.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
 # cipher
 
-> Encrypt or decrypt files on NTFS drives.
+> Encrypt or decrypt files on NTFS drives or show their encryption information.
```
Comment (CleanMachine1) [inline pages/windows/cipher.md:3]:
```suggestion
> Encrypt, decrypt files on NTFS drives, or show their encryption information.
```

File: pages/windows/cipher.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 > More information: <https://learn.microsoft.com/windows-server/administration/windows-commands/cipher>.
 
-- Encrypt a file or directory:
+- Display information about the encrypted file or directory:
```
Comment (CleanMachine1) [inline pages/windows/cipher.md:6]:
```suggestion
- Display information about a specific encrypted file or directory:
```

File: pages/windows/cipher.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
 # cipher
 
-> Encrypt or decrypt files on NTFS drives.
+> Encrypt, decrypt files on NTFS drives, or show their encryption information.
```
Comment (navarroaxel) [inline pages/windows/cipher.md:3]: Not sure about this
```suggestion
> Encrypt or decrypt files on NTFS volumes, or show their encryption information.
```

Comment (navarroaxel) [reply pages/windows/cipher.md:3]: Or based on the man page
```suggestion
> Displays or alters the encryption of directories and files on NTFS volumes.
```

Comment (lenaArtinger) [reply pages/windows/cipher.md:3]: The second one is a good general description, i guess.
Thanks for your suggestion!

**Review by kbdharun — APPROVED** [review]

**Pull request merged**